frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx

```diff
@@ -41,7 +42,11 @@ export async function action({ request }: ActionFunctionArgs) {
   const session = await sessionService.getSession(request.headers.get('Cookie'));
   const preferredLanguageSession = await session.get('preferredLanguage');
   await userService.updateUserInfo(userId, { preferredLanguage: preferredLanguageSession });
-  return redirect('/personal-information');
+  return redirectWithSuccess('/personal-information', 'gcweb:toast.preferred-language-updated', {
```

Comment:
Similar comment as above, except the i18n key should be moved within `personal-information:preferred-language.confirm`.